docs/src/components/header/HeaderNavDropdown.tsx

```diff
@@ -98,16 +98,12 @@ const DOCS = [
     description: 'Advanced and powerful components for complex use cases.',
     href: ROUTES.advancedComponents,
   },
-  // @ts-ignore
-  ...(process.env.DEPLOY_ENV === 'production'
-    ? []
-    : [
-        {
-          name: 'MUI Toolpad',
-          description: 'Low-code admin builder.',
-          href: ROUTES.toolpadDocs,
-        },
-      ]),
+  {
+    name: 'MUI Toolpad',
+    description: 'Low-code admin builder.',
+    href: ROUTES.toolpadDocs,
```

Comment:
This is a 301, it links https://mui.com/toolpad/getting-started/setup/ but this page was moved.